templates/docker-compose.template

```diff
@@ -5,40 +5,35 @@ services:
   gateway-config:
     image: nebraltd/hm-config:{{CONFIG_VERSION}}
     depends_on:
-      - helium-miner
+      - gateway-rs
       - diagnostics
-      - dbus-session
     environment:
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
       - FIRMWARE_SHORT_HASH={{ENV.FIRMWARE_SHORT_HASH}}
-      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
-      - DBUS_SESSION_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
```

Comment:
System bus is still required I believe

Reply:
Same as above. This was a thing I'd like to discuss while the PR is open.